### DISCUSSION TITLE
Question about auto-complete in Mac

### DISCUSSION THREAD

**Discussion by arjunmahishi** [neoclide/coc.nvim#2707] [Q&A]
I have configured coc for golang on my Mac. Auto-complete suggestions are showing up. But, when I press `.`, the first option in auto-complete suggestion is being selected automatically. Ideally, it should search through the suggestions as I start typing right?

(Attaching a gif for more clarity. The 3rd time I press the `.`, it automatically selects the `Errorf` function)

![gif](https://i.imgur.com/gLDQD7F.gif)

This was working fine on my linux machine. Seeing this only in Mac. Any help is appreciated. Thanks!

**Comment by chemzqm** [comment]
Looks like your have suggest.preselect set to true.

**Reply by arjunmahishi** [reply]
No I don't. This is my config:
```json
{
  "languageserver": {
    "golang": {
      "command": "gopls",
      "rootPatterns": ["go.mod", ".vim/", ".git/", ".hg/"],
      "filetypes": ["go"]
    }
  }
}
```
The `preselect` setting is `false` by default right?

**Reply by chemzqm** [reply]
`suggest.enablePreselect` in your coc-settings.json.

**Reply by chemzqm** [reply]
Use ` echo CocAction('getConfig', 'suggest')['enablePreselect']` to check it.

**Reply by arjunmahishi** [reply]
It's set to `false`. Just checked.

**Reply by arjunmahishi** [reply]
Could this be a neovim level setting?

**Reply by chemzqm** [reply]
Don't know, looks like it's not from coc.nvim.
